libpod/image/image.go

```diff
@@ -338,10 +352,24 @@ func (i *Image) TagImage(tag string) error {
 	return i.imageruntime.store.SetNames(i.ID(), tags)
 }
 
+// UnTagImage removes a tag from the given image
+func (i *Image) UnTagImage(tag string) error {
+	var newTags []string
+	tags := i.Names()
+	if util.StringInSlice(tag, tags) {
```

Comment:
This seems inverted? Don't we want to remove the tag if it's present?

Reply:
fixed